recording/lib/storage/s3.ex

```diff
@@ -56,4 +88,95 @@ defmodule Membrane.RTC.Engine.Endpoint.Recording.Storage.S3 do
     |> then(&ExAws.Config.new(:s3, &1))
     |> Map.to_list()
   end
+
+  defp objects_to_fix(objects, files) do
+    Enum.reject(files, fn {filename, {_file_path, file_size}} ->
+      s3_size_result = Map.fetch(objects, filename)
+      correct_object?(s3_size_result, file_size)
```

Comment:
suggestion: I'm not sure if `correct_object?` function is needed. 
```suggestion
      s3_size_result = Map.get(objects, filename,-1)
      s3_size_result >= file_size
```